docker-compose.yml

```diff
@@ -365,6 +365,56 @@ services:
       mode: replicated
       replicas: "${DISCOVERY_PROVIDER_REPLICAS}"
 
+  discovery-provider-test:
+    build:
+      context: discovery-provider
+      dockerfile: Dockerfile.test
```

Comment:
I think we should be able to use the existing Dockerfile and just override the command

Reply:
This would be slick

Reply:
agreed. will make this change

Reply:
done